07-multi-hop/07-multi-hop.md

```diff
@@ -1,12 +1,38 @@
 ## Goal: Check multi-hop connectivity over IPv6
 
+### Pre-setup Steps
+1. Book n>15 iotlab M3 nodes (preferably in grenoble) in a straight line 3 nodes apart. (so x--x--x--x)
+2. Add the following CFLAGS to the `tests/gnrc_udp`:
+```
+CFLAGS += -DGNRC_PKTBUF_SIZE=512 -DGNRC_NETIF_IPV6_ADDRS_NUMOF=20 \
+          -DGNRC_IPV6_NIB_NUMOF=20 \
+          -DGNRC_IPV6_NIB_OFFL_NUMOF=20
```

Comment:
I'm not sure those big numbers are really necessary.

Reply:
It works and compiles on the M3 nodes.  I usually book about 20 node and I think if these numbers are too small the routing tables can't handle it.

Reply:
Yes of course it works. More address space to allocate doesn't make it suddenly not work ;-). But I'm wondering if it is really necessary to have all that address space.